wd_helper.au3

```diff
@@ -1302,10 +1302,15 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 
 		$sBrowserVersion = _WD_GetBrowserVersion($sBrowser)
 		$iErr = @error
-		Local $iIndex = @extended
 
 		If $iErr = $_WD_ERROR_Success Then
-			; Match exe file name in list of supported browsers
+			If StringInStr($sBrowser, '\') Then
+				; Extract filename from full path
+				$sBrowser = StringRegExpReplace($sBrowser, "^.*\\|\..*$", "")
+			EndIf
+
+			; Match in list of supported browsers
+			Local $iIndex = _ArraySearch($_WD_SupportedBrowsers, $sBrowser, Default, Default, Default, Default, Default, $_WD_BROWSER_Name)
```

Comment:
I'm not understanding why this needed to be changed. Did the earlier usage of `$iIndex = @extended` not work? Please explain.

Reply:
oops ... cross version error.
Will fix this soon.

Reply:
Take a look: [StringInStr($sBrowser, '\') and $iIndex --- fixed](https://github.com/Danp2/au3WebDriver/pull/236/commits/a23eb7f278559a77ef389e3a2705628dd5b243c3)